hack/build-antrea-linux-all.sh

```diff
@@ -114,9 +114,12 @@ echo "BUILD_TAG: $BUILD_TAG"
 # new base images in the build chain.
 if $PULL; then
     if [[ ${DOCKER_REGISTRY} == "" ]]; then
+        docker pull $PLATFORM_ARG ubuntu:20.04
```

Comment:
I don't find any the following steps in the script needing ubuntu:20.04. I guess this patch fixes the test because this script runs first and caches the image on the Node, which resolves the problem but is not good for long term maintenance.
Could you point out where ubuntu:20.04 is needed? Perhaps there is a better place to pull the images.

cc @antoninbas

Reply:
this script is just to build Antrea and there is no longer a dependencies on ubuntu 20.04 for building Antrea itself.
If some tests require ubuntu 20.04 to be pulled, then the test scripts should take care of pulling the image. As Quan pointed out, this doesn't seem like the right place to do it.

Note that I am planning to upgrade the base image to ubuntu 22.04 for the rest of the container images today (Flow Aggregator, ...).

Reply:
Yes, currently flexible-ipam-e2e will fail when building Flow Aggregator image, since it will pull ubuntu:20.04 from dockerhub and will get rate limit error. If we'll update other images to ubuntu:22.04, we can retry after that PR merged.